micro-frontends/src/ipd/IpdDashboard.jsx

```diff
@@ -19,11 +18,7 @@ export function IpdDashboard(props) {
 // Without propTypes, react2angular won't render the component
 IpdDashboard.propTypes = {
   hostData: PropTypes.shape({
-    patient: PropTypes.shape({
-      uuid: PropTypes.string.isRequired,
-    }).isRequired,
-  }),
-  hostApi: PropTypes.shape({
-    onConfirm: PropTypes.func,
-  }),
+    patientId: PropTypes.string,
+    forDate: PropTypes.instanceOf(Date),
```

Comment:
Do we need to send the forDate from Bahmni Apps?
I think the navigation of dates will be driven from IPD module only. So I think sending only patientId would be sufficient.